config/default/field.field.node.article.field_image_caption.yml

```diff
@@ -10,7 +10,7 @@ field_name: field_image_caption
 entity_type: node
 bundle: article
 label: Caption
-description: 'This field is currently not rendering to visitors and will be addressed in a later update.'
+description: 'Provide a caption for this featured image. A tooltip will render on the image on the page to display the caption.'
```

Comment:
Need to update this comment based on the shift away from tooltip

Reply:
Good catch--I think if it renders directly as a caption, there's not much else to add beyond "Provide a caption for this featured image."

Reply:
Resolved by d10baf8abbc443194f31b9400b18b6983388c561